scripts-available/CDB_CartodbfyTable.sql

```diff
@@ -933,11 +964,7 @@ BEGIN
   sql := Format('CREATE TABLE %s AS SELECT ', copyname);
 
   -- Add cartodb ID!
-  IF has_usable_primary_key THEN
-    sql := sql || const.pkey;
-  ELSE
-    sql := sql || 'nextval(''' || destseq || ''') AS ' || const.pkey;
-  END IF;
+  sql := sql || 'nextval(''' || destseq || ''') AS ' || const.pkey;
```

Comment:
Phew, I almost didn't comment :) I assumed you'd only do that for some larger purpose I didn't understand :)